util/env-bootstrap.sh

```diff
@@ -0,0 +1,483 @@
+            *gentoo*) echo "app-arch/zstd app-arch/unzip app-arch/zip net-misc/wget llvm-core/clang sys-apps/hwloc dev-libs/hidapi app-text/dos2unix dev-vcs/git" ;;
+            *slackware*) echo "python3" ;;
+            *solus*) echo "system.devel zstd git wget zip unzip python3 dos2unix git" ;;
+            *void*) echo "zstd git make wget unzip zip python3 dos2unix git" ;;
+            *)
+                echo "Sorry, we don't recognize your distribution. Try using the docker image instead:" >&2
+                echo >&2
+                echo "https://docs.qmk.fm/#/getting_started_docker" >&2
+                exit 1
+                ;;
+            esac
+            ;;
+        *)
+            echo "Sorry, we don't recognize your OS. Try using a compatible OS instead:" >&2
+            echo >&2
+            echo "https://docs.qmk.fm/newbs_getting_started#set-up-your-environment" >&2
+            exit 1
+            ;;
+        esac
+    }
+
+    print_package_manager_deps_and_delay() {
+        get_package_manager_deps | tr ' ' '\n' | sort | xargs -I'{}' echo "    - {}" >&2
+        preinstall_delay || exit 1
+    }
+
+    install_package_manager_deps() {
+        # Install the necessary packages for the package manager
+        case $(fn_os) in
+        macos)
+            if [ -n "$(command -v brew 2>/dev/null || true)" ]; then
+                echo "It will also install the following system packages using 'brew':" >&2
+                print_package_manager_deps_and_delay
+                brew update && brew upgrade --formulae
+                brew install $(get_package_manager_deps)
+            else
+                echo "Please install 'brew' to continue. See https://brew.sh/ for more information." >&2
+                exit 1
+            fi
+            ;;
+        windows)
+            echo "It will also install the following packages using 'pacman'/'pacboy':" >&2
+            print_package_manager_deps_and_delay
+            $(nsudo) pacman --needed --noconfirm --disable-download-timeout -S pactoys
+            $(nsudo) pacboy sync --needed --noconfirm --disable-download-timeout $(get_package_manager_deps)
+            ;;
+        linux)
+            case $(grep ID /etc/os-release) in
+            *arch* | *manjaro*)
+                echo "It will also install the following system packages using 'pacman':" >&2
+                print_package_manager_deps_and_delay
+                $(nsudo) pacman --needed --noconfirm -S $(get_package_manager_deps)
+                ;;
+            *debian* | *ubuntu*)
+                echo "It will also install the following system packages using 'apt':" >&2
+                print_package_manager_deps_and_delay
+                $(nsudo) apt-get update
+                DEBIAN_FRONTEND=noninteractive \
+                    $(nsudo) apt-get --quiet --yes $(get_package_manager_deps)
```

Comment:
Operation seems missing (should be `apt-get --quiet --yes install).

Else apt complains w/ `E: Invalid operation zstd`.

Reply:
Good catch. Updated.